### ISSUE TITLE
[fixed error] aws-sdk remoteprovider httprequest timeouterror on wsl vscode gatsby aws-amplify

### ISSUE THREAD

**Issue by ywroh** [aws/aws-sdk-js#3410]
Confirm by changing [ ] to [x] below:
- [x] I've gone through [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/welcome.html) and [API reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/)
- [x] I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-js) for answers

**Describe the question**

I am working on deploying react gatsby based ecommerce site to aws amplify.

The development environment is being developed in the wsl vscode environment.

When you input the gatsby develop command in the project, a timeouterror occurred in aws-sdk remoteprovider httprequest.

Add config and credential files to ~/.aws directory, execute wsl --shutdown command in windows powershell, set node debugg in vscode and turn it off, timeouterror disappeared.

In conclusion, the problem is solved, but I am curious about why this problem occurred and how it was solved, so I leave it as an issue.

Thank you for always providing good information.

**Comment by ajredniwja** [comment]
Hey @ywroh thanks for opening this issue, however I wont be able to say anything without getting into details. 

I am going to close this for now, I can answer the questions if you can provide me with some reproduction steps.

**Comment by ywroh** [comment]
Thank you for your answer.

Currently, the same error is not occurring.

If the same error occurs next time, we will register the issue with specific details.
